app/views/manager/events/_form.html.erb

```diff
@@ -14,6 +14,10 @@
           <label for="description" class="block mb-2 text-sm font-medium text-gray-900 dark:text-white"><%= t('activerecord.attributes.event.description') %></label>
```

Comment:
Refatore o i18n, entenda sobre: https://dev.to/risafj/the-basics-of-rails-i18n-translate-errors-models-and-attributes-84d